modules/afsocket/afinet-dest.c

```diff
@@ -564,6 +559,8 @@ afinet_dd_is_spoof_source_enabled(AFInetDestDriver *self)
   return self->spoof_source && self->lnet_ctx;
 }
 
+#endif
```

Comment:
Optional: I like to keep code chunks behind compiler switches as minimal as possible. If the functions are small, then both the function and the compiler switch is visible. Otherwise it is easy to overlook if I am under a compiler switch or not. So I would keep the original one.